### PR TITLE
Keycloak

### PR DESCRIPTION
**Pull request by edewit**
initial wrapper around http that will enable SSO

### CODE REVIEW

**Comment by gastaldi** [comment]
This just adds the wrapper, it doesn't change anything specific in the UI, right?

File: src/assets/keycloak/keycloak.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (gastaldi) [inline src/assets/keycloak/keycloak.json]: Is this bundled with the frontend image?

Comment (edewit) [inline src/assets/keycloak/keycloak.json]: yes, see see
https://github.com/obsidian-toaster/generator-frontend/pull/36/files#diff-f737fedb5357468044db2dbe98eb0218R37

**Comment by edewit** [comment]
Right, it doesn't do anything in the UI atm. When you where already logged in it will add the token to the request to the backend. We could create a login button or a check in the UI in future as well.

**Pull request merged**